client/src/components/NewGame/NewGame.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from 'react'
+import classnames from 'classnames'
+
+import './NewGame.scss'
+
+import Button from '../Button/Button'
+
+export interface NewGameProps {
+  clickHandler: (event: React.MouseEvent<HTMLButtonElement>) => void
```

Comment:
I would rename `clickHandler` to `createGame`, also the type should be `() => void`, as per the signature of the createGame function you're passing from Home.tsx

Reply:
Ok, but I don't really understand `() => void`. I will find some readings about this!